crates/languages/src/python.rs

```diff
@@ -39,6 +41,22 @@ use std::{
 use task::{ShellKind, TaskTemplate, TaskTemplates, VariableName};
 use util::{ResultExt, maybe};
 
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct PythonToolchainData {
+    #[serde(flatten)]
+    environment: PythonEnvironment,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    activation_scripts: Option<HashMap<ShellKind, PathBuf>>,
+}
+
+impl Deref for PythonToolchainData {
```

Comment:
I don't really love implicit Derefs; could we instead just bite a bullet and access the `.environment` directly?

Reply:
Oh sure thing!